src/utils/uiUtils.ts

```diff
@@ -57,6 +58,20 @@ export async function promptForSignIn(): Promise<void> {
     }
 }
 
+export async function promptHintMessage(config: string, message: string, choiceConfirm: string, onConfirm: () => Thenable<any>): Promise<void> {
+    if (getWorkspaceConfiguration().get<boolean>(`hint.${config}`)) {
```

Comment:
Just pass the entire `hint.${config}` to the method instead of only the `${config}` itself since we may have other setting section in the future.